src/main/java/io/qameta/allure/bamboo/AllureBuildCompleteAction.java

```diff
@@ -1,41 +1,44 @@
 package io.qameta.allure.bamboo;
 
 import com.atlassian.bamboo.build.BuildDefinition;
-import com.atlassian.bamboo.chains.Chain;
 import com.atlassian.bamboo.chains.ChainExecution;
 import com.atlassian.bamboo.chains.ChainResultsSummary;
 import com.atlassian.bamboo.chains.plugins.PostChainAction;
 import com.atlassian.bamboo.configuration.AdministrationConfiguration;
+import com.atlassian.bamboo.plan.cache.ImmutableChain;
 import com.atlassian.bamboo.resultsummary.ResultsSummary;
 import com.atlassian.bamboo.resultsummary.ResultsSummaryManager;
 import com.atlassian.bamboo.v2.build.BaseConfigurablePlugin;
 import com.atlassian.spring.container.ContainerManager;
+import com.google.gson.Gson;
+import com.google.gson.JsonParser;
 import io.qameta.allure.bamboo.info.AddExecutorInfo;
+import io.qameta.allure.bamboo.info.allurewidgets.summary.Summary;
+import io.qameta.allure.bamboo.util.Downloader;
+import io.qameta.allure.bamboo.util.FileStringReplacer;
+import io.qameta.allure.bamboo.util.ZipUtil;
 import org.apache.commons.lang3.StringUtils;
 import org.jetbrains.annotations.NotNull;
 import org.slf4j.Logger;
 import org.slf4j.LoggerFactory;
 
-import java.io.File;
-import java.io.IOException;
-import java.io.InputStream;
-import java.net.HttpURLConnection;
+import java.io.*;
```

Comment:
Please don't use combination if imports:

Checks that there are no import statements that use the * notation.

Rationale: Importing all classes from a package or static members from a class leads to tight coupling between packages or classes and might lead to problems when a new version of a library introduces name clashes.